README.md

```diff
@@ -1,65 +1,109 @@
 # HTTPRequest
 
-HTTPRequest is a single-header library for making HTTP requests. You can just include it in your project and use it. HTTPRequest was tested on macOS, Windows and Linux, but it should work on most of the Linux-based platforms. Supports IPv4 and IPv6.
+HTTPRequest is a single-header library for making HTTP requests. You can just include it in your project and use it.
+| :heavy_check_mark: Supports IPv4 and IPv6 |
+| --- |
+
+# HTTPRequest was tested on
+- MacOS
+- Windows
+- Linux
```

Comment:
If you are listing operating systems and not kernels, then use "GNU/Linux" instead of "Linux"

Reply:
This has been taken from the current README.md "HTTPRequest was tested on macOS, Windows and Linux, but it should work on most of the Linux-based platforms."

I will change it to "GNU / Linux".